examples/notebooks/beam-ml/run_inference_pytorch_tensorflow_sklearn.ipynb

```diff
@@ -0,0 +1,1349 @@
+        "outputId": "2a22059f-519c-44f7-e36f-59e09b1cb24a"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stderr",
+          "text": [
+            "WARNING:tensorflow:From /usr/local/lib/python3.7/dist-packages/tfx_bsl/beam/run_inference.py:615: load (from tensorflow.python.saved_model.loader_impl) is deprecated and will be removed in a future version.\n",
+            "Instructions for updating:\n",
+            "This function will only be available through the v1 compatibility library as tf.compat.v1.saved_model.loader.load or tf.compat.v1.saved_model.load. There will be a new function for importing SavedModels in Tensorflow 2.0.\n",
+            "WARNING:apache_beam.io.tfrecordio:Couldn't find python-snappy so the implementation of _TFRecordUtil._masked_crc32c is not as fast as it could be.\n"
+          ]
+        },
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "input is [[b'this is such an amazing movie']] output is [0.99906057]\n",
+            "input is [[b'The movie was great']] output is [0.99307914]\n",
+            "input is [[b'The movie was okish']] output is [0.03274685]\n",
+            "input is [[b'The movie was terrible']] output is [0.00680008]\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### RunInference with Scikit-Learn\n"
+      ],
+      "metadata": {
+        "id": "8wBUckzHjGV6"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#### Install Dependency"
+      ],
+      "metadata": {
+        "id": "6ArL_55kjxkO"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install --upgrade pip\n",
+        "!pip install apache_beam[gcp]==2.41.0"
+      ],
+      "metadata": {
+        "id": "R4p6Mil0jxSy"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import pickle\n",
+        "\n",
+        "import apache_beam as beam\n",
+        "from apache_beam.ml.inference import RunInference\n",
+        "from apache_beam.ml.inference.sklearn_inference import SklearnModelHandlerNumpy, ModelFileType"
+      ],
+      "metadata": {
+        "id": "_YtRRxh1hLag"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#### Model\n",
+        "\n",
+        "Train and save a sentiment analysis pipeline on movie reviews to classify movie reviews as either positive or negative"
+      ],
+      "metadata": {
+        "id": "-7ABKlZvkFHy"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Train model based on this [tutorial](https://scikit-learn.org/stable/tutorial/text_analytics/working_with_text_data.html#exercise-2-sentiment-analysis-on-movie-reviews)"
```

Comment:
Sorry I missed this earlier; could we do the same thing we do above with Tensorflow and just give a link to a gcs bucket and say "This model was trained by following this [tutorial](https://scikit-learn.org/stable/tutorial/text_analytics/working_with_text_data.html#exercise-2-sentiment-analysis-on-movie-reviews)"